### PR TITLE
Update icon

### PR DESCRIPTION
**Pull request by maxb2**
I updated the icon with a better font that resembles the one on the front page. I also removed all the old, unused icons.

dr-icon

### CODE REVIEW

**Comment by n1ru4l** [comment]
I created those icons using https://realfavicongenerator.net/ for all platforms. Why do you think those are unused? 😅 Also you could try https://jakearchibald.github.io/svgomg/ for slimming down the SVG. I also think that the `R` should have more padding to the right.

**Comment by maxb2** [comment]
> Why do you think those are unused?

We don't [reference](https://github.com/dungeon-revealer/dungeon-revealer/search?q=icon&type=Code) those files anywhere in the project :sweat_smile: I figured they were unused since your rewrite a long time ago.

The favicon has the following embedded in it:

```
dr-icon.ico[0] PNG 256x256 256x256+0+0 16-bit sRGB 29769B 0.000u 0:00.000
dr-icon.ico[1] ICO 192x192 192x192+0+0 8-bit sRGB 0.000u 0:00.000
dr-icon.ico[2] ICO 128x128 128x128+0+0 8-bit sRGB 0.000u 0:00.000
dr-icon.ico[3] ICO 96x96 96x96+0+0 8-bit sRGB 0.000u 0:00.000
dr-icon.ico[4] ICO 64x64 64x64+0+0 8-bit sRGB 0.000u 0:00.000
dr-icon.ico[5] ICO 48x48 48x48+0+0 8-bit sRGB 0.000u 0:00.000
dr-icon.ico[6] ICO 40x40 40x40+0+0 8-bit sRGB 0.000u 0:00.000
dr-icon.ico[7] ICO 32x32 32x32+0+0 8-bit sRGB 0.000u 0:00.000
dr-icon.ico[8] ICO 24x24 24x24+0+0 8-bit sRGB 0.000u 0:00.000
dr-icon.ico[9] ICO 16x16 16x16+0+0 8-bit sRGB 328871B 0.000u 0:00.000
```

> Also you could try https://jakearchibald.github.io/svgomg/ for slimming down the SVG.

Will do!

> I also think that the R should have more padding to the right.

I was going for "center of mass" rather than geometric center. It's less pronounced for the smaller icons. That's easy enough to change though.

**Comment by maxb2** [comment]
Here is a version with the text geometrically centered. You're right, it's better.
dr-favicon-2

**Comment by n1ru4l** [comment]
Yeah my fault I actually did this in another project and mixed things up in my mind 😂. It would be cool if you could generate cross platform da icons with and also link those in the index.html https://realfavicongenerator.net/

**Comment by maxb2** [comment]
@n1ru4l can you test the iOS home screen icons? I already tested it for android.

**Comment by n1ru4l** [comment]
IMG_A1190EC9F450-1

Minified SVG:

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 135.46666 135.46667" height="512" width="512"><defs><path id="b" d="M71.935437 38.142081h141.44605v125.32152H71.935437z"/><path id="a" d="M23.687553 87.907883h146.68129v102.04227H23.687553z"/></defs><rect ry="10.253784" y="15.233963" x=".41729647" height="104.99874" width="134.63208" fill="#006900" stroke="#006900" stroke-width=".834593"/><g fill="#fff" font-weight="400" font-family="EB Garamond SC"><text style="line-height:1.25;-inkscape-font-specification:'EB Garamond SC, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;white-space:pre;shape-inside:url(#a)" transform="matrix(1.53222 0 0 1.7841 -33.1593512 -153.9219102)" font-size="64.4673"><tspan x="23.6875" y="142.76988"><tspan style="-inkscape-font-specification:'EB Garamond SC, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal">D</tspan></tspan></text><text style="line-height:1.25;-inkscape-font-specification:'EB Garamond SC, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;white-space:pre;shape-inside:url(#b)" transform="matrix(1 0 0 1.1644 -8.7482277 -41.60024837)" font-size="98.7778"><tspan x="71.935547" y="122.20249"><tspan style="-inkscape-font-specification:'EB Garamond SC, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal">R</tspan></tspan></text></g></svg>
```
https://jakearchibald.github.io/svgomg/

**Comment by maxb2** [comment]
Since we're not serving the svg anywhere, I left it as the editor version (and didn't put it in the public folder). I figured it would make future edits easier. I can add an "optimized" version too. Another thing to note, https://realfavicongenerator.net/ didn't render the font correctly in the original svg. I had to convert the font to a path before it worked correctly.

**Comment by maxb2** [comment]
@n1ru4l this is ready to merge once you have completed your review.

File: public/images/icons/site.webmanifest [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (n1ru4l) [inline public/images/icons/site.webmanifest:1]: I think we should serve this file on the root instead in the images directory

File: public/images/icons/site.webmanifest [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "name": "Dungeon Revealer App",
+    "short_name": "Dungeon Revealer",
+    "icons": [
+        {
+            "src": "/public/images/icons/android-chrome-192x192.png",
```
Comment (n1ru4l) [inline public/images/icons/site.webmanifest:6]: The /public path is incorrect and can be omitted.

File: server/server.js [hunk 3]
```diff
@@ -48,7 +48,7 @@ const bootstrapServer = async () => {
   // Not sure if this is needed, Chrome seems to grab the favicon just fine anyway
   // Maybe for cross-browser support
   app.use(logger("dev"));
-  app.use(favicon(path.resolve(env.PUBLIC_PATH, "favicon.ico")));
+  app.use(favicon(path.resolve(env.PUBLIC_PATH, 'images/icons', "favicon.ico")));
```
Comment (n1ru4l) [inline server/server.js:51]: https://nodejs.org/api/path.html#path_path_resolve_paths

ˋ'images', 'icons',ˋ Is the better solution as then the platform conform separator will be used.

**Pull request merged**